spec/selection.spec.js

```diff
@@ -294,6 +294,66 @@ describe('Selection', function () {
 
   })
 
+  describe('triming:', function () {
+    beforeEach(function () {
```

Comment:
add a case when only a whitespace is selected

Reply:
Good idea. What would you expect to happen if a selection only consists of whitespace (one or more)? IMO we should then just leave it alone and not trim anything. Fine for you?